share/completions/cdrecord.fish

```diff
@@ -47,7 +47,7 @@ complete -c cdrecord -o setdropts -d "Set driver specific options"
 complete -c cdrecord -o checkdrive -d "Checks if a driver for the current drive is present"
 complete -c cdrecord -o prcap -d "Print the drive capabilities for SCSI-3/mmc compliant drives"
 complete -c cdrecord -o inq -d "Do an inquiry for the drive, print the inquiry info"
-complete -c cdrecord -o scanbus -d "Scan all SCSI devices on all SCSI busses and print the inquiry strings"
+complete -c cdrecord -o scanbus -d "Scan all SCSI devices on all SCSI buses and print the inquiry strings"
```

Comment:
This one is ok both ways too - https://www.merriam-webster.com/grammar/plural-of-bus

Reply:
Both are valid, so keeping the existing wording or using the one I suggest are valid.

I would the then tend to avoid having diff in git history for nothing, and I agree with not updating it

But in the exact article you mentioned, you can read

> The plural of the noun bus is buses. You might see the plural busses, but that form is so rare that it seems like an error to many people.

So I would like to keep the change I'm introducing here

Reply:
going with "buses"